website/docs/d/scc_posture_scan_summaries.html.markdown

```diff
@@ -0,0 +1,108 @@
+			* `controls_total_count` - (Optional, Integer) The total number of controls that were included in the scan.
+	* `group_profiles` - (Optional, List) The result of a group profile.
+	Nested scheme for **group_profiles**:
+		* `group_profile_id` - (Optional, String) The group ID of profile.
+		* `group_profile_name` - (Optional, String) The group name of the profile.
+		* `profile_type` - (Optional, String) The type of profile. To learn more about profile types, check out the [docs] (https://cloud.ibm.com/docs/security-compliance?topic=security-compliance-profiles).
+		  * Constraints: Allowable values are: standard, authored, custom, standard_cv, temmplategroup, standard_certificate
+		* `validation_result` - (Optional, List) The result of a scan.
+		Nested scheme for **validation_result**:
+			* `goals_pass_count` - (Optional, Integer) The number of goals that passed the scan.
+			* `goals_unable_to_perform_count` - (Optional, Integer) The number of goals that could not be validated. A control is listed as 'Unable to perform' when information about its associated resource can't be collected.
+			* `goals_not_applicable_count` - (Optional, Integer) The number of goals that are not relevant to the current scan. A scan is listed as 'Not applicable' when information about its associated resource can't be found.
+			* `goals_fail_count` - (Optional, Integer) The number of goals that failed the scan.
+			* `goals_total_count` - (Optional, Integer) The total number of goals that were included in the scan.
+			* `controls_pass_count` - (Optional, Integer) The number of controls that passed the scan.
+			* `controls_fail_count` - (Optional, Integer) The number of controls that failed the scan.
+			* `controls_not_applicable_count` - (Optional, Integer) The number of controls that are not relevant to the current scan. A scan is listed as 'Not applicable' when information about its associated resource can't be found.
+			* `controls_unable_to_perform_count` - (Optional, Integer) The number of controls that could not be validated. A control is listed as 'Unable to perform' when information about its associated resource can't be collected.
+			* `controls_total_count` - (Optional, Integer) The total number of controls that were included in the scan.
+		* `profiles` - (Optional, List) The result of a each profile in group profile.
+		Nested scheme for **profiles**:
+			* `profile_id` - (Optional, String) The ID of the profile.
+			* `profile_name` - (Optional, String) The name of the profile.
+			* `profile_type` - (Optional, String) The type of profile. To learn more about profile types, check out the [docs] (https://cloud.ibm.com/docs/security-compliance?topic=security-compliance-profiles).
+			  * Constraints: Allowable values are: standard, authored, custom, standard_cv, temmplategroup, standard_certificate
```

Comment:
If constraints part of profilte_type, we need to add Nested scheme..... and
Supported values are .... the values should be marked as bold